.github/workflows/publish.yml

```
@@ -0,0 +1,43 @@
+name: Version and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      newversion:
+        description: 'Version'
+        required: true
+
+concurrency:
+  group: npm-bump
+  cancel-in-progress: true
+
+jobs:
+  version_and_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        registry-url: 'https://registry.npmjs.org'
+
+    - uses: pnpm/action-setup@v2
+      id: pnpm-install
+      with:
+        version: 7
+
+    - run: pnpm install
+
+    - run: pnpm test
+
+    - uses: bcomnes/npm-bump@v2
+      with:
+        git_email: bcomnes@gmail.com
+        git_username: ${{ github.actor }}
```

I think we can just associate `github-actions` with this, otherwise this email may be confusing or look suspicious. 

```suggestion
        git_email: github-actions@github.com
        git_username: github-actions
```